tests/zenko_tests/node_tests/iam_policies/cloudserver/AssumeRoleWithWebIdentity.js

```diff
@@ -0,0 +1,144 @@
+const assert = require('assert');
+const async = require('async');
+const VaultClient = require('../../VaultClient');
+const { makeGETRequest } = require("../../utils/request");
+const { getS3Client} = require("../../s3SDK");
+const { getSTSClient} = require("../../stsSDK");
+const { getTokenForIdentity } = require("../../utils/getWebIdentityToken");
+
+let iamClient = null;
+let stsClient = null;
+let s3Client = null;
+
+const clientAdmin = VaultClient.getAdminClient();
+const accountName = 'AccountTest';
+const accountInfo = {
+    email: `${accountName}@test.com`,
+    password: 'test',
+};
+const externalAccessKey = 'DZMMJUPWIUK8IWXRP0HQ';
+const externalSecretKey = 'iTuJdlidzrLipymvAGrLP66Yxghl4NQxLZR3cLlu';
+
+const duration = '1000';
+
+const bucket1 = "bucket1";
+const bucket2 = "bucket2";
+
+const storageManagerName = 'storage_manager';
+const storageAccountOwnerName = 'storage_account_owner';
+const dataConsumerName = 'data_consumer';
+const storageManagerRole = 'storage-manager-role';
+const storageAccountOwnerRole = 'storage-account-owner-role';
+const dataConsumerRole = 'data-consumer-role';
+
+describe('iam policies - cloudserver - AssumeRoleWithWebIdentity - Metadata', () => {
+
+    before(done => {
+        async.series([
+            // create an account, generateAccountAccessKey for it, get iam client, sts client and s3 client of this account
+            next => clientAdmin.createAccount(accountName, accountInfo, next),
+            next => clientAdmin.generateAccountAccessKey(accountName, next, { externalAccessKey, externalSecretKey }),
```

Comment:
Have discussed with William privately, `ZENKO_ACCESS_KEY` is the storage_manager_role user of an account named `zenko`, I can't use this user to assumeRoleWithWII of the 3 default roles of account `zenko`. Because we can see from the[ keycloak configure file](https://github.com/scality/Zenko/blob/development/2.3/eve/workers/end2end/scripts/configs/keycloak_config.json#L6), there are only 3 default users defined in keycloak under the account `AccountTest`, so I have to create a new account named `AccountTest` to be able to use the 3 existing users in keycloak.